Quaver.Shared/Database/Playlists/PlaylistManager.cs

```diff
@@ -513,6 +522,62 @@ public static void UpdateMapInPlaylists(Map outdated, Map newMap)
             });
         }
 
+        /// <summary>
+        ///     Imports an online playlist
+        /// </summary>
+        /// <param name="onlineId"></param>
+        public static void ImportPlaylist(int onlineId)
+        {
+            Logger.Important($"Importing playlist: ID {onlineId}", LogType.Runtime);
+            var playlistResponse = new APIRequestPlaylistInformation(onlineId).ExecuteRequest();
```

Comment:
Shouldn't this be wrapped in try/catch, or is there some higher level try/catch?

Reply:
Should be added. It will crash if the request fails.